src/org/omegat/gui/exttrans/MachineTranslateTextArea.java

```diff
@@ -147,18 +205,34 @@ private void startSearchThread(SourceTextEntry newEntry, boolean force) {
     @Override
     protected void setFoundResult(final SourceTextEntry se, final MachineTranslationInfo data) {
         UIThreadsUtil.mustBeSwingThread();
-
         if (data != null && data.result != null) {
-            displayed.add (data);
-            setText(getText() + data.result + "\n<" + data.translatorName + ">\n\n");
+            String indexString = String.format("%d", index);
```

Comment:
This is an extremely expensive way of stringifying an int (`String#valueOf` would be better), and it's not even necessary: you can simply do `StringBuilder#append` with the int and the result will be the same.

Reply:
I change here to use `StringBuilder#append`.